### PR TITLE
 * NEW [mqtt/all] MQTT V5 : set Maximum Packet Size for broker & send…

### PR DESCRIPTION
**Pull request by JaylinYu**

### CODE REVIEW

File: src/sp/transport/mqtt/broker_tcp.c [hunk 1]
```diff
@@ -510,14 +510,16 @@ tcptran_pipe_recv_cb(void *arg)
...
 		// Make sure the message payload is not too big.  If it is
 		// the caller will shut down the pipe.
-		// if ((len > p->rcvmax) && (p->rcvmax > 0)) {
-		// 	debug_msg("size error\n");
-		// 	rv = NNG_EMSGSIZE;
-		// 	goto recv_error;
-		// }
+		if (len > NANO_MAX_RECV_PACKET_SIZE) {
+			debug_msg("size error 0x95\n");
+			rv = 0x95;
```
Comment (wanghaEMQ) [inline src/sp/transport/mqtt/broker_tcp.c:515]: Better with macro

File: src/sp/transport/mqtt/broker_tcp.c [hunk 2]
```diff
@@ -510,14 +510,16 @@ tcptran_pipe_recv_cb(void *arg)
...
+			//send disconnect to client by rpaio;
+			goto recv_error;
+		}
 
 		if ((rv = nni_msg_alloc(&p->rxmsg, (size_t) len)) != 0) {
 			debug_syslog("mem error %ld\n", (size_t) len);
+			rv = 136;
```
Comment (wanghaEMQ) [inline src/sp/transport/mqtt/broker_tcp.c:522]: Here also

**Review by wanghaEMQ — CHANGES_REQUESTED** [review]
Macros are defined in file nng/include/nng/protocol/mqtt/mqtt.h

File: src/sp/transport/mqtt/broker_tcp.c (hunk 2 above)
Comment (JaylinYu) [reply src/sp/transport/mqtt/broker_tcp.c:522]: okok

**Comment by JaylinYu** [comment]
let's have a look again.

**Pull request merged**